packages/examples/packages/send-flow/src/index.test.tsx

```diff
@@ -232,9 +300,19 @@ describe('onUserInput', () => {
 
     expect(updatedInterface3).toRender(
       <SendFlow
-        accounts={accountsArray}
-        selectedAccount={accountsArray[1].address}
-        selectedCurrency="BTC"
+        account={{
+          accountId: MOCK_ACCOUNTS[1].id,
+          addresses: [
+            `${MOCK_ACCOUNTS[1].scopes[0]}:${MOCK_ACCOUNTS[1].address}`,
+          ],
+        }}
+        asset={{
+          asset: MOCK_ACCOUNTS[1].assets[0],
+          name: 'Solana',
+          symbol: 'SOL',
+        }}
```

Comment:
Would be nicer if we could just do something like

```suggestion
        account={MOCK_ACCOUNTS[1]}
        asset={MOCK_ACCOUNTS[1].assets[0]}
```

Reply:
The thing is that the component is supposed to receive the interface state here, i could create utils function to create the state from the mocks

Reply:
Something like this? I think that's ok too.

```suggestion
        account={getMockAccount(MOCK_ACCOUNTS[1])}
        asset={getMockAsset(MOCK_ACCOUNTS[1].asset[0])}
```